sample/allofhands_hawaii2016/tracks

```diff
@@ -0,0 +1,450 @@
+		"location": null,
+		"sessions": [{
+				"id": 75,
+				"title": "IT: Infosec IR Process"
+			},
+			{
+				"id": 92,
+				"title": "IT: Cloud Video Conferencing RFP Results"
+			},
+			{
+				"id": 138,
+				"title": "IT: IT All Hands"
+			},
+			{
+				"id": 152,
+				"title": "IT: EUS Goes MIG"
+			},
+			{
+				"id": 25,
+				"title": "IT: Systems Team"
+			},
+
+			{
+				"id": 157,
+				"title": "IT: App Services: SN Dev Team"
+			},
+			{
+				"id": 63,
+				"title": "IT: Document Vendor Procurement Process"
+			},
+			{
+				"id": 23,
+				"title": "IT: EUS Training Workshops"
+			},
+			{
+				"id": 74,
+				"title": "IT: Commons AV System Design"
+			},
+			{
+				"id": 26,
+				"title": "IT: WebOps Q1 2017 Planning Session"
+			},
+			{
+				"id": 24,
+				"title": "IT: MOC Incident Management - Understanding Prioritization"
+			},
+
+			{
+				"id": 10,
+				"title": "IT: Application Services - Service Strategy Pivot"
+			},
+			{
+				"id": 225,
+				"title": "IT: Service Desk Looking Forward"
+			},
+			{
+				"id": 229,
+				"title": "IT: WebOps 2016 in Review"
+			},
+			{
+				"id": 11,
+				"title": "IT: Infosec 2017 Planning"
+			},
+			{
+				"id": 65,
+				"title": "IT: Security Awareness 2017"
+			},
+			{
+				"id": 207,
+				"title": "IT: Data Team Working Session"
+			}
+		],
+		"track_image_url": null
+	},
+	{
+		"id": 4,
+		"name": "Strategic Ops",
+		"color": "#9EFF56",
+		"description": null,
+		"location": null,
+		"sessions": [{
+				"id": 209,
+				"title": "WPR\\, Metrics & Internal Comms Homeroom"
+			},
+			{
+				"id": 45,
+				"title": "Electives Coaching - Final Session"
+			},
+			{
+				"id": 52,
+				"title": "WPR Insights Session"
+			},
+			{
+				"id": 102,
+				"title": "Strategic Operations All hands"
+			},
+			{
+				"id": 130,
+				"title": "WPR\\, Metrics & Internal Comms Homeroom"
+			}
+		],
+		"track_image_url": null
+	},
+	{
+		"id": 5,
+		"name": "people",
+		"color": "#E8782C",
+		"description": null,
+		"location": null,
+		"sessions": [{
+				"id": 13,
+				"title": "Ops OKR Presentation / Project Planning"
+			},
+			{
+				"id": 95,
+				"title": "Business Processes: Part 1"
+			},
+			{
+				"id": 208,
+				"title": "Business Processes: Part 2"
+			},
+			{
+				"id": 51,
+				"title": "EE Experience"
+			},
+			{
+				"id": 77,
+				"title": "Merit Cycle Process Conversation"
+			},
+			{
+				"id": 129,
+				"title": "People Ops planning"
+			},
+
+			{
+				"id": 226,
+				"title": "Employment Law 101"
+			}
+
+		],
+		"track_image_url": null
+	},
+	{
+		"id": 6,
+		"name": "FP&A",
+		"color": "#FF3046",
+		"description": null,
+		"location": null,
+		"sessions": [{
+				"id": 22,
+				"title": "FP&A Session with Jim"
+			},
+			{
+				"id": 216,
+				"title": "FP&A Session with Jim"
+			},
+			{
+				"id": 59,
+				"title": "FP&A Session with Jim"
+			},
+			{
+				"id": 137,
+				"title": "FP&A Session with Jim"
+			}
+		],
+		"track_image_url": null
+	},
+	{
+		"id": 7,
```

Comment:
This track does not conform to the actual data. This contains only 2 sessions, whereas there are several sessions under marketing. 15 to be exact. Please check that all data is correctly linked